jetty-jaspi/src/main/java/module-info.java

```diff
@@ -16,7 +16,7 @@
     requires jetty.servlet.api;
     requires org.slf4j;
 
-    requires transitive javax.security.auth.message;
+    requires transitive java.security.auth.message;
```

Comment:
Are you sure that's right? Jaspi is `javax.security.auth.messge`.

Reply:
There is no `module-info.java` in `jakarta.security.auth.message-api-1.1.3.jar` and in the `META-INF/MANIFEST.MF` it says:
```
Automatic-Module-Name: java.security.auth.message
```


You cannot build the project if you use `javax.security.auth.messge` or `jakarta.security.auth.messge`. It says:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.11.0:compile (default-compile) on project jetty-jaspi: Compilation failure
[ERROR] /home/lachlan/webtide/jetty-10.0.x/jetty-jaspi/src/main/java/module-info.java:[19,44] module not found: javax.security.auth.message
```

Reply:
Groan ... that's  a mess.